### DISCUSSION TITLE
Use SVG-Edit with load and save from server

### DISCUSSION THREAD

**Discussion by tobei** [SVG-Edit/svgedit#657] [Q&A]
Hi there, 

I intend to use SVG-Edit to edit svg slides in a website i'm making. For that purpose I should be able to instruct SVG-Edit to open an URL containing the image, and once the user is finished editing it, the save button should post the edited image back to that URL (or any other) on my server so I can update it in my backend.

In practice for the loading I found the following that seems to do the charm: 
`svgEditor.loadFromURL(...)`
Is it the preferred way to do?

But for saving, pushing the save buttons still default to opening a local file system dialog.
In the documentation i saw those config parameters:

`

      avoidClientSideDownload: false,

      avoidClientSideOpen: false`

But they don't seem to have any effect? Or are they used by an extension that is not part of SVG-Edit packaging? How to make my editor actually prefer working with URLS for download and upload? 

Documentation also points to a custom handler for save event but the Editor no longer have a setCustomHandler function
`svgEditor.setCustomHandlers({
  save (_win, _data) {
    // Save svg
  }
});
`

PS: i'm using the v7 freshly built

Thanks a lot !

**Comment by jfhenon** [comment]
Hello ,
Indeed, V7 works with the local filesystem by default (even though you can open from an URL). 
In order to change this behavior, we would need to adapt the fileopen extension. Please look at the code https://github.com/SVG-Edit/svgedit/blob/master/src/editor/extensions/ext-opensave/ext-opensave.js.
If you issue a PR, we can add it to the list of default extensions.
Thanks
JF
